provision.sh

```diff
@@ -41,7 +43,7 @@ KONG_ADMIN_LISTEN_SSL="0.0.0.0:8444"
 
 if [ $KONG_NUM_VERSION -gt 001003 ]; then
   #Kong 0.10.4 and later are on Bintray
-  KONG_DOWNLOAD_URL="https://bintray.com/kong/kong-community-edition-deb/download_file?file_path=dists%2Fkong-community-edition-${KONG_VERSION}.trusty.all.deb"
+  KONG_DOWNLOAD_URL="https://bintray.com/kong/kong-community-edition-deb/download_file?file_path=dists%2Fkong-community-edition-${KONG_VERSION}.bionic.all.deb"
```

Comment:
Yes they are: https://bintray.com/kong/kong-community-edition-deb/ubuntu/0.12.0#files/dists

Reply:
tested this and it didn't work, because we only build Bionic packages from 0.13. So older versions fail in the step to download Kong.

Just pushed another commit to fix it.